xml/Microsoft.Extensions.Logging.AzureAppServices.Internal/BlobAppendReferenceWrapper.xml

```diff
@@ -27,32 +27,24 @@
   </Docs>
   <Members>
     <Member MemberName=".ctor">
-      <MemberSignature Language="C#" Value="public BlobAppendReferenceWrapper (Microsoft.Azure.Storage.Blob.CloudAppendBlob cloudAppendBlob);" />
-      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(class Microsoft.Azure.Storage.Blob.CloudAppendBlob cloudAppendBlob) cil managed" />
-      <MemberSignature Language="DocId" Value="M:Microsoft.Extensions.Logging.AzureAppServices.Internal.BlobAppendReferenceWrapper.#ctor(Microsoft.Azure.Storage.Blob.CloudAppendBlob)" />
+      <MemberSignature Language="C#" Value="public BlobAppendReferenceWrapper (Microsoft.WindowsAzure.Storage.Blob.CloudAppendBlob cloudAppendBlob);" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor(class Microsoft.WindowsAzure.Storage.Blob.CloudAppendBlob cloudAppendBlob) cil managed" />
+      <MemberSignature Language="DocId" Value="M:Microsoft.Extensions.Logging.AzureAppServices.Internal.BlobAppendReferenceWrapper.#ctor(Microsoft.WindowsAzure.Storage.Blob.CloudAppendBlob)" />
       <MemberSignature Language="VB.NET" Value="Public Sub New (cloudAppendBlob As CloudAppendBlob)" />
-      <MemberSignature Language="F#" Value="new Microsoft.Extensions.Logging.AzureAppServices.Internal.BlobAppendReferenceWrapper : Microsoft.Azure.Storage.Blob.CloudAppendBlob -&gt; Microsoft.Extensions.Logging.AzureAppServices.Internal.BlobAppendReferenceWrapper" Usage="new Microsoft.Extensions.Logging.AzureAppServices.Internal.BlobAppendReferenceWrapper cloudAppendBlob" />
+      <MemberSignature Language="F#" Value="new Microsoft.Extensions.Logging.AzureAppServices.Internal.BlobAppendReferenceWrapper : Microsoft.WindowsAzure.Storage.Blob.CloudAppendBlob -&gt; Microsoft.Extensions.Logging.AzureAppServices.Internal.BlobAppendReferenceWrapper" Usage="new Microsoft.Extensions.Logging.AzureAppServices.Internal.BlobAppendReferenceWrapper cloudAppendBlob" />
       <MemberSignature Language="C++ CLI" Value="public:&#xA; BlobAppendReferenceWrapper(Microsoft::WindowsAzure::Storage::Blob::CloudAppendBlob ^ cloudAppendBlob);" />
       <MemberType>Constructor</MemberType>
       <AssemblyInfo>
         <AssemblyName>Microsoft.Extensions.Logging.AzureAppServices</AssemblyName>
         <AssemblyVersion>1.0.0.0</AssemblyVersion>
       </AssemblyInfo>
       <Parameters>
-        <Parameter Name="cloudAppendBlob" Type="Microsoft.Azure.Storage.Blob.CloudAppendBlob" Index="0" FrameworkAlternate="dotnet-plat-ext-1.1" />
+        <Parameter Name="cloudAppendBlob" Type="Microsoft.WindowsAzure.Storage.Blob.CloudAppendBlob" Index="0" FrameworkAlternate="dotnet-plat-ext-1.1" />
       </Parameters>
       <Docs>
-        <param name="cloudAppendBlob">The <see cref="T:Microsoft.Azure.Storage.Blob.CloudAppendBlob" /> instance to wrap.</param>
```

Comment:
There's some data loss here.

Reply:
@gewarren  
public BlobAppendReferenceWrapper (Microsoft.**WindowsAzure**.Storage.Blob.CloudAppendBlob cloudAppendBlob); 
is a new method and it matches the source code.
![image](https://user-images.githubusercontent.com/73150173/217131526-fad33b95-576d-414d-a8c4-0a927926d2cd.png)
The old method is deleted with its documentation.

Reply:
Okay, I had changed the namespace in https://github.com/dotnet/dotnet-api-docs/pull/8746 to fix the warnings here: https://github.com/dotnet/dotnet-api-docs/pull/8745#issuecomment-1366967885. But it looks like those warnings are gone now anyway. I'll make a commit here to retain the documentation comments.